packages/metastream-app/src/containers/LobbyPage.tsx

```diff
@@ -180,6 +180,10 @@ export class _LobbyPage extends Component<PrivateProps, IState> {
     this.disconnect(reason, true)
   }
 
+  private reconnect = () => {
+    this.setupLobby();
+  }
```

Comment:
Since pressing the Retry button will immediately show the connecting screen, I don't think disabling the button will be needed.